### PR TITLE
Dor hakim

### LINKED ISSUE #36: Add style to "today"
# Description 
Add a highlight to today in the day picker to make it easier to find.
Try to use some shadow or border, nothing too fancy.

# Example
(Don't actually implement it like that)
image

### PR DESCRIPTION
**Pull request by dorhakim100**
Closes #36

Added a new class with a unique style for today.
Checking whether or not provided date is today, if so add today class.

### CODE REVIEW

File: src/features/views/day/day-carousel/DayItem.module.scss [hunk 1]
```diff
@@ -14,6 +14,8 @@
     }
   }
 
+  // DorHakim
```
Comment (Polarts) [inline src/features/views/day/day-carousel/DayItem.module.scss:17]: You don't need to put your name everywhere. Git shows which changes you made.
Please remove all ` // DorHakim` comments.

File: src/features/views/day/day-carousel/DayItem.tsx [hunk 2]
```diff
@@ -30,10 +30,20 @@ const DayItem = ({ date, active, onClick }: DayItemProps) => {
...
 
-    return { dayLetter, dayNumber, count, invalid, mappedRating }
+    // DorHakim
+    const today = isToday(date)
+
+    // Or implementing ourselves
+    // const today =
+    //   date.toLocaleDateString('he') === new Date().toLocaleDateString('he')
+    //     ? true
+    //     : false
```
Comment (Polarts) [inline src/features/views/day/day-carousel/DayItem.tsx:40]: If we were to implement it ourselves, it's not a good implementation.
`toLocaleDateString('he')` creates a string specifically in Hebrew. Why would we want to compare strings, and evben ones with Hebrew characters, when we can compare numbers efficiently?
A better implementation would be comparing `getYear`, `getMonth` and `getDate`.
Either way, the library handles it so you may delete the commented code.

**Review by Polarts — CHANGES_REQUESTED** [review]

**Comment by Polarts** [comment]
@nick-vi Run the [preview](https://sentispace-txtn-git-fork-dorhakim100-dorhakim-polarts-projects.vercel.app/) and tell me what you think about the design

**Comment by nick-vi** [comment]
I like the subtle font change, but the drop shadow doesn’t align with the design language. Please remove the shadow, and we’ll be ready to merge.

**Review by Polarts — APPROVED** [review]

**Pull request merged**